snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsTypes/CS/keywordsTypes.cs

```diff
@@ -98,46 +98,39 @@ int y
           get;
           set;
        }
+
+       public double Distance 
```

Comment:
I think access modifiers shouldn't be specified in an interface.

```suggestion
       double Distance 
```